src/tools/rbd/action/Export.cc

```diff
@@ -86,12 +86,13 @@ class C_Export : public Context
   SimpleThrottle &m_throttle;
   librbd::Image &m_image;
   bufferlist m_bufferlist;
+  uint64_t f_offset;
```

Comment:
The "m_" prefix is for instance variables -- if you want to distinguish this as the destination offset, name it "m_dest_offset"